style/bs4_book.html

```diff
@@ -90,12 +90,15 @@ <h2>Table of contents</h2>
 
   <div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
     <h2>Note: Second Edition is under construction 🏗</h2>
-    <p>Now is a great time to test in-development packages and provide feedback</p>
+    <p>Now is a great time to provide feedback</p>
         <ul class="list-unstyled">
+          <li><a href="https://twitter.com/Ukraine/status/1497294708879540224">Support Ukraine 🇺🇦
           <li><a href="https://forms.gle/nq9RmbxJyZXQgc948">Provide feedback (5 min)</a></li>
           <li><a href="https://geocompr.robinlovelace.net/#reproducibility">Install updated packages</a></li>
           <li><a href="https://github.com/Robinlovelace/geocompr/issues">Open an issue <i class="fas fa-question"></i></a></li>
           <li><a href="https://discord.gg/Te3gWeDwmf">Chat on Discord <i class="fab fa-discord"></i></a></li>
+          <li><a href="https://twitter.com/Ukraine/status/1497294708879540224">Support Ukraine 🇺🇦
```

Comment:
Why did you put the same link a 2nd time?

Reply:
Was to emphasise it but in hindsight once is enough. Only 1 line now.